<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcExplorer</class>
 <widget class="QWidget" name="ProcExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="bProc">
       <property name="text">
        <string>proc/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bKill">
       <property name="text">
        <string>Kill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bScan">
       <property name="text">
        <string>Scn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bBundle">
       <property name="text">
        <string>Bundle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bPostProc">
       <property name="text">
        <string>Postproc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
